### ISSUE TITLE
oni.input is undefined

### ISSUE THREAD

**Issue by MikaAK** [extr0py/oni#710]
I've just installed Oni and I'm trying to unbind `ctrl-p` however inside activate callback `oni.input` is undefined.

The following errors with `oni.input is undefined`
```javascript
const activate = (oni) => {
   console.log("config activated")

   oni.input.unbind("<c-p>")
}

const deactivate = () => {
   console.log("config deactivated")
}

module.exports = {
  activate,
  deactivate,
  "oni.loadInitVim": true,
  "oni.useDefaultConfig": false,
  "editor.fontFamily": "Hack"
}
```

**Comment by extr0py** [comment]
Hi @MikaAK ! Thanks for logging the issue.

Which version of Oni are you using? The `input` API is included in `0.2.9`, which I just started building release candidates for. You might want to try the latest rc build.

I also updated the Wiki with a note that version 0.2.9 is required for this API.

**Comment by MikaAK** [comment]
Ah moving to the RC did indeed fix this. Sorry for the confusion. I'm super excited to start using this, as a vim user it's really frustrating to not be able to use the cool new IDE features from things like typescript! Good work 😄  Hope I can contribute one day
